vendor/github.com/vmware/govmomi/toolbox/command.go

```diff
@@ -472,19 +473,22 @@ func (c *CommandServer) ListFiles(header vix.CommandRequestHeader, data []byte)
 		return nil, err
 	}
 
-	info, err := os.Stat(r.GuestPathName)
+	info, err := os.Lstat(r.GuestPathName)
```

Comment:
For these kinds of changes we will either need to make a PR request against govmomi or request that doug possibly make them in the govmomi. @dougm thoughts?